arch/src/x86_64/mod.rs

```diff
@@ -410,6 +411,15 @@ pub fn configure_vcpu(
             .map_err(|_| Error::PopulatingCpuid)?;
     }
 
+    // Override CPU physical bits
+    if let Some(phys_bits) = phys_bits {
+        for entry in cpuid.as_mut_slice().iter_mut() {
+            if entry.function == 0x8000_0008 {
+                entry.eax = (entry.eax & 0xffff_ff00) | (phys_bits as u32 & 0xff);
```

Comment:
I'm wondering if we should somehow check here or earlier the specified value is not larger than the host can support. The provides a small safety net in case people specify too large a value.

(This is not a blocking comment.)

Reply:
For example, in the CI patch you specify the value to 40. I wonder what happens if I run that locally on a desktop class machine whose physical address size is 39 bits.

Reply:
I just tried setting the value to 40 on a host with 32 bits physical. Windows still boots. Interesting.

Reply:
I think this one is for correctness purposes, there's nothing making it mandatory for the guest to read this leaf.